server/src/db.js

```diff
@@ -5,19 +5,18 @@ import {Client} from 'pg'
 export const db = new Client(c.knex.connection)
 db.connect()
 
-export async function loadReportsByTag(tag) {
+export async function loadReports() {
   return (await db.query(
     `SELECT ts, "user", message, channel, response_to
-     FROM tag LEFT JOIN report ON tag.report=report.ts
-     WHERE tag.tag = $1
-     ORDER BY ts ASC`,
-    [tag],
+     FROM report
```

Comment:
Do I understand correctly, that there are reports in the database that were never read before? Can you check and confirm this to me, please? I am might have misunderstood it, but it looks to me, that every comment is saved in the database, regardless if it has a tag or not.

If that's the case, we don't need to load all the reports here. It's unnecessary. You would need to change the original query to something like:
```
SELECT ts, tag.tag, "user", message, channel, response_to
     FROM tag LEFT JOIN report ON tag.report=report.ts
     ORDER BY ts ASC
```

So, only the tagged reports are loaded.

Reply:
Yes, untagged messages are also stored in the db.